src/components/Card/CardBody.tsx

```diff
@@ -30,9 +33,18 @@ const cardBody = ({ children, sectioned, theme, componentStyle, componentClass }
   );
 
   return (
-    <div className={className} style={componentStyle}>
-      {bodyContent}
-    </div>
+    <Context.Consumer>
+    {
+      context =>
+        <div
+          className={className}
+          style={componentStyle}
+          onClick={(e) => { (context.cardHasOnClick && onClick) && e.stopPropagation(); onClick && onClick(e); }}
```

Comment:
Please create onClick handler function instead of writing all functionality here